bundler/bundler.go

```diff
@@ -58,7 +58,7 @@ func bundle(model *v3.Document, inline bool) ([]byte, error) {
 			//if we're in the root document, don't bundle anything.
 			refExp := strings.Split(sequenced.FullDefinition, "#/")
 			if len(refExp) == 2 {
-				if refExp[0] == "" {
+				if refExp[0] == sequenced.Index.GetSpecAbsolutePath() {
```

Comment:
Sometimes (depending on the configuration) there may or may not be a full path, it may be empty (which is why the original logic stands). 

```suggestion
				if refExp[0] == sequenced.Index.GetSpecAbsolutePath() || refExp[0] == "" {
```